plugins/woocommerce-admin/client/marketplace/components/product-list-content/product-list-content.tsx

```diff
@@ -1,19 +1,40 @@
 /**
  * External dependencies
  */
+import { useContext } from '@wordpress/element';
+import { Spinner } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
+import { ProductListContext } from '../../contexts/product-list-context';
 import './product-list-content.scss';
 
 export default function ProductListContent(): JSX.Element {
+	const productListContextValue = useContext( ProductListContext );
```

Comment:
I reckon it would be nice to call this const and the one in `Search` the same thing – would `productListContext` work as an instance of `ProductListContext`?